saleor/graphql/order/tests/test_order.py

```diff
@@ -5830,6 +5831,94 @@ def test_orders_query_with_filter_search_by_id_with_hash(
     assert content["data"]["orders"]["totalCount"] == 1
 
 
+def test_orders_query_with_filter_search_by_product_sku_with_allocation(
+    orders_query_with_filter, staff_api_client, permission_manage_orders, allocations
+):
+    variables = {"filter": {"search": allocations[0].order_line.product_sku}}
+    staff_api_client.user.user_permissions.add(permission_manage_orders)
+    response = staff_api_client.post_graphql(orders_query_with_filter, variables)
+    content = get_graphql_content(response)
+    assert content["data"]["orders"]["totalCount"] == 3
+
+
+def test_order_query_with_filter_search_by_product_sku_order_line(
+    orders_query_with_filter, staff_api_client, permission_manage_orders, order_line
+):
+    variables = {"filter": {"search": order_line.product_sku}}
+    staff_api_client.user.user_permissions.add(permission_manage_orders)
+    response = staff_api_client.post_graphql(orders_query_with_filter, variables)
+    content = get_graphql_content(response)
+    assert content["data"]["orders"]["totalCount"] == 1
+
+
+def test_order_query_with_filter_search_by_product_sku_multi_order_lines(
+    orders_query_with_filter,
+    staff_api_client,
+    permission_manage_orders,
+    product,
+    channel_USD,
+    order,
+):
+    first_variant = ProductVariant.objects.create(product=product, sku="Var1")
```

Comment:
Everywhere, where you create multiple objects of one type, you should always use (bulk)[https://docs.djangoproject.com/en/3.2/ref/models/querysets/#bulk-create].
Also, you could consider, just extend `test_orders_query_with_filter_search`,  instead of building a new test only  for this param